packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -542,7 +542,8 @@ export class ConnectionManager implements IConnectionManager {
 			return;
 		}
 
-		let delayMs = InitialReconnectDelayInMs;
+		// Initialize with half as when we calculate the actual wait time, then we double it.
+		let delayMs = InitialReconnectDelayInMs / 2;
```

Comment:
I'd probably not do it here (i.e. not devide here). But instead reduce InitialReconnectDelayInMs and mention where it's defined that a value of double will be used (and point to the only place where it happens )